spec/models/user_spec.rb

```diff
@@ -44,45 +44,60 @@
 describe User do
```

Comment:
I would add a test for when omniauth is not successful and throws an error

Reply:
Yes, I was thinking about that too. Gonna work on that.